### ISSUE TITLE
Export Selected Rows Not Working in Dojo GridX

### ISSUE THREAD

**Issue by ghost** [oria/gridx#420]
I am exploring Gridx control in Dojo. I am curious about Export and print module of GridX. While checking [exporter ](http://oria.github.io/gridx/demos/exporter.html) example I found that '**Selected Rows Only**' functionality does not work. It **exports all rows of grid irrespective of what I selected**.Has anybody faced this issue ? How can I implement 'export selected rows only' functionality in GridX? Is it a bug with GridX Export module ?

Ref Javascript https://github.com/oria/gridx/blob/master/tests/test_grid_exporter.js

**Comment by rsrini7** [comment]
Yes. I am also exploring. after filter only the export is restricted. "Selected rows - export" not working
